SAMv1.tex

```diff
@@ -367,12 +367,10 @@ \subsubsection{Reference MD5 calculation}\label{sec:ref-md5}
 and the corresponding tag is {\tt M5:caad65b937c4bc0b33c08f62a9fb5411}.
 
 \subsection{The alignment section: mandatory fields}\label{sec:alnrecord}
-In the SAM format, each alignment line typically represents the linear
-alignment of a segment. Each line has 11
-mandatory fields. These fields always appear in the same order and must be
-present, but their values can be `0' or `*' (depending on the field) if the
-corresponding information is unavailable. The following table gives an overview
-of the mandatory fields in the SAM format:
+In the SAM format, each alignment line typically represents the linear alignment of a segment.
+Each line has eleven mandatory fields.
+These fields always appear in the same order and must be present, though their values may be `{\tt 0}' or `{\tt *}' (depending on the field type) when the corresponding information is unavailable.
```

Comment:
This makes it sounds like the values could also be something else when the information is unavailable....perhaps reword as 
> These fields always appear in the same order and must be present when available.
When unavailable, their values must be `{\tt 0}' or `{\tt *}' (depending on the field type).

Reply:
That's the original Heng paragraph, basically unchanged. What it actually wants is a larger rewrite! I'll merge the main part of this PR and the OA tag and then come back to it…